src/types/decimal/coefficient.rs

```diff
@@ -17,67 +18,6 @@ pub enum Sign {
     Positive,
 }
 
-/// A simple wrapper type around a [`UInt`]. Users are not expected to construct
-/// instances of `Magnitude` directly; it is provided for conversion ergonomics.
-///
-/// Signed integer types cannot implement `Into<UInt>`. Instead, they implement
-/// `TryInto<UInt>` and report an error if the input integer is negative.
-///
-/// Signed integers can infallibly implement `Into<Magnitude>` by using their
-/// absolute value.
-pub struct Magnitude(UInt);
```

Comment:
🗺️ This type was moved into its own module. You'll see its definition appear further on.